### ISSUE TITLE
Add notes download?

### ISSUE THREAD

**Issue by ansibleforsap** [sap-linuxlab/community.sap_launchpad#21]
Hi, would it be possible to add the ability to download SNOTES from the Support Portal? I have an ansible playbook that needs to get the latest compatible OS-release for the respective HANA version (Note 2235581)

**Comment by sean-freeman** [comment]
@ansibleforsap Yes it would be possible, but the contribution will have to come from the community. See discussion in #15 

_NOTE: I would recommend you change your handle...._

**Comment by ansibleforsap** [comment]
Hmm i see. I am afraid i lack the neccessary skills to implement what you described :/

What's up with my handle? O.o

**Comment by sean-freeman** [comment]
We can leave this open, I've added a label to do the GH Issue.

Your handle implies you are part of the initiative team, and now you have opened a GH Issue within the GH Org - your handle will appear when people raise new Issues etc.  Changing the handle would avoid that potential outcome, which would potentially spam your email address and also might cause us to miss an end-users help request / comment.
